contracts/core/interfaces/ICore.sol

```diff
@@ -125,6 +125,17 @@ interface ICore {
     )
         external;
 
+    /**
+     * Adds or removes a price library to the mapping of tracked price libraries. Can only be set by
+     * owner of Core.
+     *
+     * @param  _priceLibrary   The address of the PriceLibrary to enable/disable
+     */
+    function setPriceLibrary(
```

Comment:
Why are we exposing this/other setters again? Not sure we ever need to expose the owner-only setters. I think this adds to the runtime gas cost by having them all in a single core interface. @bweick, we should do an exercise on that and confirm. If true, we should definitely have logic specific interfaces that expose 1-2 methods at a time.